### ISSUE TITLE
The content of README.md does not reside at rocm.docs.amd.com

### ISSUE THREAD

**Issue by atamazov** [ROCmSoftwarePlatform/MIOpen#2596]
Old documentation has the "Build and Install Instructions" section https://rocm.github.io/MIOpen/doc/html/install.html#build-and-install-instructions.

The new documentation is missing it, see https://rocm.docs.amd.com/projects/MIOpen/en/latest/index.html. 

The source of the missing documentation is README.md. Missing sections:
```
### How to build documentation
## Building MIOpen from source
### Configuring with cmake
### Setting Up Locations
### System Performance Database and User Database
### Persistent Program Cache
### Changing the cmake configuration
## Building the library
## Building the driver
## Running the tests
## Formatting the code
## Storing large file using Git LFS
## Installing the dependencies manually
## Using docker
## Porting from cuDNN to MIOpen
```

----
[Attribution] @junliume @JehandadKhan
- https://github.com/ROCm/MIOpen/labels/bug
- https://github.com/ROCm/MIOpen/labels/documentation
- https://github.com/ROCm/MIOpen/labels/urgency_blocker
- Proposed assignees: @xu-shawn

**Comment by Rmalavally** [comment]
@bghimireamd 

We have the build and install RST files in the updated ToC for ROCm 6.0 at 

https://github.com/ROCm/MIOpen/blob/Rmalavally-patch-1/docs/sphinx/_toc.yml.in

Is the content different from these files? Please advise. @JehandadKhan @atamazov

**Comment by atamazov** [comment]
@Rmalavally 
> We have the build and install RST files in the updated ToC for ROCm 6.0 at https://github.com/ROCm/MIOpen/blob/Rmalavally-patch-1/docs/sphinx/_toc.yml.in

[Important] Does this mean that we have **_two_** copies of the documentation in the source tree?

> Is the content different from these files?

Unfortunately I do not fully understand the question (what _content_?) But it seems like the problem described in this issue persists in [the files you are mentioning](https://github.com/ROCm/MIOpen/blob/Rmalavally-patch-1/docs/sphinx/_toc.yml.in) as well.

**Comment by Rmalavally** [comment]
Sorry, I was not clear. 

We do not have two copies of the documentation in the source tree. I moved the RST files to the new folder structure.

We have _build_, _install,_ and _embed_ RST files in the updated table of contents. Are you suggesting changes to these files?

**Comment by atamazov** [comment]
@Rmalavally 
> We do not have two copies of the documentation in the source tree. I moved the RST files to the new folder structure.

I see, thanks.

> We have build, install, and embed RST files in the updated table of contents. 

Unfortunately I am not aware of the source tree structure you are using and how it is mapped to the actual TOC etc. That's why I'm not talking in terms of individual files, I'm talking about the content. The main thing is that the information from README.md is not lost, while in which files it should be put is a secondary question.

**Comment by Rmalavally** [comment]
Thank you, Artem, for your detailed feedback. It is much appreciated. 

Please let me know which time zone you are in, and I am happy to schedule a quick meeting with you. I can walk you through the existing structure vs. the proposed changes. 

@bghimireamd is familiar with the changes we are making. I will also discuss this with him (IST) this evening my time (CA, PST)  and try to ensure the Readme changes are captured in the documentation. Given the initial MIOpen documentation restructure was planned for ROCm 6.0 three days ago, we may be constrained in time for the release.

**Comment by junliume** [comment]
Hi @Rmalavally please keep me, @JehandadKhan also in the loop. We'd hope that information is enhanced and at least not lost during the restructure.

**Comment by Rmalavally** [comment]
For sure, @junliume. I prefer working with SMEs for any documentation update for this reason. Appreciate your, Jehandad, and the team's timely help.

**Comment by ppanchad-amd** [comment]
@Rmalavally Is the documentation fixed in latest ROCm 6.1.0? Thanks!

**Comment by Rmalavally** [comment]
> @Rmalavally Is the documentation fixed in latest ROCm 6.1.0? Thanks!

As of February 2024, yes.

**Comment by ppanchad-amd** [comment]
@atamazov Can we close this ticket? Thanks!

**Comment by Rmalavally** [comment]
> > @Rmalavally Is the documentation fixed in latest ROCm 6.1.0? Thanks!
> 
> As of February 2024, yes.

Here's the link - https://rocm.docs.amd.com/projects/MIOpen/en/latest/index.html
